### PR TITLE
MaskedCorners feature

### PR DESCRIPTION
**Pull request by wixvii**
This PR helps to add masked corners for additionView corners on SwipeMenuView

### CODE REVIEW

**Comment by yysskk** [comment]
Hi, @wixvii . Thanks to PR 😄 
I'd like to take in this `maskedCorners` 👍

File: Sources/Classes/SwipeMenuView.swift [hunk 1]
```diff
@@ -62,6 +66,9 @@ public struct SwipeMenuViewOptions {
             
             /// AdditionView animating duration. Defaults to `0.3`.
             public var animationDuration: Double = 0.3
+            
+            /// AdditionView swipe animation disable feature Defaults to 'true'
+            public var animationOnSwipe: Bool = true
```
Comment (yysskk) [inline Sources/Classes/SwipeMenuView.swift]: Please rename to `isAnimationOnSwipeEnable`

File: Sources/Classes/TabView.swift [hunk 2]
```diff
@@ -361,6 +361,35 @@ extension TabView {
             additionView.layer.position.y = itemView.layer.position.y
             additionView.layer.cornerRadius = options.additionView.circle.cornerRadius ?? additionView.frame.height / 2
             additionView.backgroundColor = options.additionView.backgroundColor
+            
+                        if #available(iOS 11.0, *) {
```
Comment (yysskk) [inline Sources/Classes/TabView.swift]: Fix indents 👀

File: Sources/Classes/TabView.swift [hunk 3]
```diff
@@ -417,21 +446,25 @@ extension TabView {
...
+                }
+            case .reverse:
+                additionView.frame.origin.x = previousItem.frame.origin.x + (currentItem.frame.origin.x - previousItem.frame.origin.x) * ratio + options.additionView.padding.left
+                additionView.frame.size.width = previousItem.frame.size.width + (currentItem.frame.size.width - previousItem.frame.size.width) * ratio - options.additionView.padding.horizontal
+                if options.needsConvertTextColorRatio {
+                    previousItem.titleLabel.textColor = options.itemView.selectedTextColor.convert(to: options.itemView.textColor, multiplier: ratio)
+                    currentItem.titleLabel.textColor = options.itemView.textColor.convert(to: options.itemView.selectedTextColor, multiplier: ratio)
+                }
             }
+        }else {
```
Comment (yysskk) [inline Sources/Classes/TabView.swift]: Please space

**Comment by yysskk** [comment]
Update README too 🙏

**Comment by wixvii** [comment]
Please verify and merge. Thanks.

**Comment by yysskk** [comment]
LGTM 🚀

**Pull request merged**